api/billingd/client/client.go

```diff
@@ -37,20 +38,32 @@ func (c *Client) CheckHealth(ctx context.Context) error {
 	return err
 }
 
-func (c *Client) CreateCustomer(ctx context.Context, key thread.PubKey, opts ...Option) (string, error) {
+func (c *Client) CreateCustomer(
+	ctx context.Context,
+	key thread.PubKey,
+	email string,
+	accountType mdb.AccountType,
+	opts ...Option,
+) (string, error) {
 	args := &options{}
 	for _, opt := range opts {
 		opt(args)
 	}
-	var parentKey string
+	var parent *pb.CreateCustomerRequest_Params
 	if args.parentKey != nil {
-		parentKey = args.parentKey.String()
+		parent = &pb.CreateCustomerRequest_Params{
```

Comment:
Parent will be created if it's missing, which means we need all the parent info to do that.